testkit/src/main/scala/org/bitcoins/testkit/core/gen/PSBTGenerators.scala

```diff
@@ -205,6 +206,41 @@ object PSBTGenerators {
     }
   }
 
+  def psbtWithBuilderAndP2SHOutputs(
+      finalized: Boolean,
+      outputGen: CurrencyUnit => Gen[Seq[(TransactionOutput, ScriptPubKey)]] =
+        TransactionGenerators.smallP2SHOutputs)(
+      implicit ec: ExecutionContext): Gen[
+    Future[(PSBT, BitcoinTxBuilder, Seq[ScriptPubKey])]] = {
+    for {
+      (creditingTxsInfo, outputs) <- CreditingTxGen.inputsAndP2SHOutputs(
```

Comment:
nit: rather than calling `outputs._1` and `outputs._2` below you could expand it to `(_, _)` here

Reply:
I tried doing that and it gives me errors, not sure if possible